dev/io.openliberty.goingpostal.1.0.internal/bnd.bnd

```diff
@@ -0,0 +1,55 @@
+#*******************************************************************************
+# Copyright (c) 2025 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+
+-sub: *.bnd
+
+bVersion=1.0
+
+src: src, resources
+
+Import-Package: \
+    javax.xml.bind.annotation;version=!,\
+    org.jboss.jandex;version="[2.4,4)",\
+    *
+    
+-dsannotations: \
+  io.openliberty.goingpostal.servlet.filter.HeaderFilterContainerInitializer
+  
+Export-Package: \
+	io.openliberty.goingpostal.servlet.filter	
```

Comment:
This should be 
`io.openliberty.goingpostal.servlet.filter.ClacksOverheadFilter.java`